### ISSUE TITLE
Bitbucket cloud v1 API deprecated

### ISSUE THREAD

**Issue by nilseuropa** [sshaw/export-pull-requests#19]
https://developer.atlassian.com/cloud/bitbucket/deprecation-notice-v1-apis/

**Comment by sshaw** [comment]
Yes thanks. Seem that these sorts of issues [have been brought to the attention of the Bitbucket API client used](https://github.com/bitbucket-rest-api/bitbucket/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+2.0) to no avail. 

PRs are using 2.0, but [issues are using 1.0](https://github.com/bitbucket-rest-api/bitbucket/blob/2fc8fa575ca3636ee8603a4b0efce95bbd79fab0/lib/bitbucket_rest_api/issues.rb#L85). 

Any suggestions for an alternate Bitbucket client?

**Comment by nilseuropa** [comment]
nope sry...
